rfc-119-merge-mainstream-browse-topics-and-taxons.md

```diff
@@ -0,0 +1,183 @@
+- `visible_to_departmental_editors`
+
+Added from mainstream browse:
+- `browse_groups` (the curated groupings of browse children)
+
+Added from specialist topic:
+- `specialist_topic_groups` (the curated groupings of specialist_topic_children)
+
+Dropped from mainstream browse:
+- `internal_name` (redundant)
+- `second_level_ordering` (can be deduced by following link to parent)
+- `ordered_second_level_browse_pages` (can be deduced by following link to parent)
+
+Dropped from specialist topics:
+- `internal_name` (redundant)
+
+### Example of proposed taxon content item after merging:
+```json
+{
+  "title": "Universal Credit",
+  "description": "Applying, signing into your account, and help with housing, disability, health conditions and unemployment",
+  "base_path": "/welfare/universal-credit",
+  "content_id": "62fcbba5-3a75-4d15-85a6-d8a80b03d57c",
+  "document_type": "taxon",
+  "schema_name": "taxon",
+  "publishing_app": "content-tagger",
+  "rendering_app": "collections",
+  "locale": "en",
+  "phase": "live",
+  "first_published_at": "2018-03-08T16:38:00.000+00:00",
+  "public_updated_at": "2018-08-22T13:14:40.000+00:00",
+  "updated_at": "2019-08-09T20:07:42.761Z",
+  "publishing_request_id": "12821-1535466331.861-10.3.3.1-394",
+  "publishing_scheduled_at": null,
+  "scheduled_publishing_delay_seconds": null,
+  "analytics_identifier": null,
+  "withdrawn_notice": {},
+  "links": {
+    "parent_taxons": [],
+    "browse_children": [],
+    "browse_parent": [],
+    "specialist_topic_children": [],
+    "specialist_topic_parent": [],
+    "primary_publishing_organisation": [],
+    "available_translations": []
+  },
+  "details": {
+    "internal_name": "Universal Credit [M]",
+    "notes_for_editors": "",
+    "visible_to_departmental_editors": false,
+    "specialist_topic_groups": [],
+    "browse_groups": [],
+    "browse_ordering": "curated",
+    "ordered_second_level_browse_pages": []
+  }
+}
+```
+
+We would probably approach this work in two stages by merging mainstream browse *and then* specialist topics. After both are merged we'd look at deprecating mainstream browse and specialist topics content items in the Publishing API and Content Store.
+
+To do this we'd change application dependencies to read from taxons. Application dependencies we'd have to consider:
+
+- Rendering `/browse/*` and `/topic/*` pages in Collections Frontend
+- Displaying metrics for `/browse/*` and `/topic/*` pages in Content Data
+- Signing up and email subscriptions per specialist topic
+- Finding browse links in the Knowledge Graph
+
+Once we are confident we are solely using the taxons, we can unpublish the existing mainstream browse and specialist topic content items.
+
+This work should not introduce any user facing changes.
+
+## Benefits
+This would allow us to ensure ongoing consistency of our "Information Architecture" and grouping of content, without the complexity of maintaining three document types or seperate sets of content items.
```

Comment:
Are there further benefits than this? I feel like this is probably solving some problems that someone is experiencing that aren't called out?

I'm wondering that whilst these things still exist as 3 separate items item on the frontend whether we're introducing limited benefits and instead risks and confusion. For example, someone might not understand that editing a taxon name actually changes a mainstream browse page and a specialist topic? Or that removing an item from the topic taxonomy could break a whole branch of a mainstream browse.

There's probably also some risks that should be called out with these